_src/static/sass/base/_reset.scss

```diff
@@ -37,18 +37,13 @@ video {
 body {
     color: $text-base-color;
     background: $view-bg;
-    font-size: $font-scale-medium;
+    font-size: font-size(base);
     font-family: $font-family-sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-.hd-1,
```

Comment:
NOTE: clean up from headings element styling - this is not needed and duplicates rules in ``_headings.scss``.